syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -573,6 +573,11 @@ def share(self, *owners, field=None, crypto_provider=None):
             .wrap()
         )
 
+    def share_(self, *args, **kwargs):
```

Comment:
Can you add a comment saying this is inplace(a la share)

Reply:
Done!